frontend/package.json

```diff
@@ -26,17 +26,17 @@
     "verify": "yarn build && yarn lint && yarn test"
   },
   "dependencies": {
-    "@openshift/dynamic-plugin-sdk": "^2.0.1",
+    "@openshift/dynamic-plugin-sdk": "^3.0.0",
     "@openshift/dynamic-plugin-sdk-extensions": "^1.1.0",
     "@openshift/dynamic-plugin-sdk-utils": "^1.1.1",
-    "@patternfly/patternfly": "^4.185.1",
-    "@patternfly/quickstarts": "^2.3.1",
-    "@patternfly/react-catalog-view-extension": "^4.53.16",
-    "@patternfly/react-core": "^4.202.16",
-    "@patternfly/react-table": "^4.71.16",
-    "@redhat-cloud-services/frontend-components": "^3.8.10",
-    "@redhat-cloud-services/frontend-components-notifications": "^3.2.14",
-    "@redhat-cloud-services/frontend-components-utilities": "^3.2.14",
+    "@patternfly/patternfly": "^5.0.2",
```

Comment:
Can we bump these to latest:
```
"@patternfly/patternfly": "^5.0.4",
"@patternfly/react-core": "^5.0.1",
"@patternfly/react-table": "^5.0.1",
```
Not required as we have the carat, but if we're pushing another commit it would be nice

Reply:
For what it's worth @florkbr and @rohitkrai03,
We did just have a PF release last week:

It addresses some UI bugs and introduces a few new features ([release notes](https://www.patternfly.org/get-started/release-highlights#patternfly-5.1))
 
"@patternfly/patternfly": "5.1.0",
"@patternfly/react-charts": "7.1.1",
"@patternfly/react-code-editor": "5.1.1",
"@patternfly/react-core": "5.1.1",
"@patternfly/react-icons": "5.1.1",
"@patternfly/react-styles": "5.1.1",
"@patternfly/react-table": "5.1.1",
"@patternfly/react-tokens": "5.1.1".
"@patternfly/react-topology": "5.1.0"

Reply:
Updated to latest.